rest-api/test/unit_test/dao_test/physical_measurements_dao_test.py

```diff
@@ -93,6 +94,8 @@ def testInsert_rightParticipantId(self):
     summary = ParticipantSummaryDao().get(self.participant.participantId)
     self.assertEquals(PhysicalMeasurementsStatus.COMPLETED, summary.physicalMeasurementsStatus)
     self.assertEquals(TIME_2, summary.physicalMeasurementsTime)
+    self.assertEquals(summary.biospecimenFinalizedSiteId, None)
```

Comment:
Same as above... there's no reason to think this field would have been set.

What we need to do is add a test that checks that we don't re-pair the participant based on physical measurements once a biobank order has been received and this field *is* set.

Reply:
new test added

Reply:
OK I think you can remove this line if you want.

Reply:
done